dashboard/config/application.rb

```diff
@@ -37,6 +37,12 @@ class Application < Rails::Application
 
     config.middleware.insert_after Rails::Rack::Logger, VarnishEnvironment
     config.middleware.insert_after VarnishEnvironment, FilesApi
+
+    if CDO.throttle_data_apis
```

Comment:
I guess what I mean is that we don't run it automatically.  If running `bin/dashboard-server` also started Redis, it might be more reasonable to just enable this in all environments.